### PR TITLE
Fixes source tags

### PR DESCRIPTION
**Pull request by jmagak**
**Version(s):** 1.2

**Issue:** https://issues.redhat.com/browse/RHIDP-3484

**Link to docs preview:**

**Reviews:**
- [ ] SME: @ mention assignee
- [ ] QE: @ mention assignee
- [ ] Docs review: @ mention assignee

**Additional information:**

### CODE REVIEW

**Review by nickboldt — APPROVED** [review]

**Comment by nickboldt** [comment]
Related issues:
* https://issues.redhat.com/browse/RHIDP-3206 (for automation)
* https://issues.redhat.com/browse/RHIDP-3484 (for manual fix)

**Comment by themr0c** [comment]
Your pull request should go to https://github.com/redhat-developer/red-hat-developers-documentation-rhdh rather than to your fork.

**Comment by themr0c** [comment]
`oc` `kubectl` `helm` are terminal commands.

File: modules/admin/proc-using-azure-auth-provider.adoc [hunk 1]
```diff
@@ -95,7 +95,7 @@ upstream:
 . Optional: If the Helm Chart is already installed, upgrade it:
 +
 --
-[source,subs="attributes+"]
+[source,yaml,subs="attributes+"]
```
Comment (nickboldt) [inline modules/admin/proc-using-azure-auth-provider.adoc]:
```suggestion
[source,terminal,subs="attributes+"]
```

File: modules/installation/proc-rhdh-deploy-aks-operator.adoc [hunk 2]
```diff
@@ -81,7 +81,7 @@ spec:
 . To deploy the created Ingress, run the following command:
 +
 --
-[source]
+[source,yaml]
```
Comment (nickboldt) [inline modules/installation/proc-rhdh-deploy-aks-operator.adoc]:
```suggestion
[source,terminal]
```

File: modules/installation/proc-rhdh-deploy-aks-operator.adoc [hunk 3]
```diff
@@ -150,7 +150,7 @@ spec:
 . Apply the CR manifest to your namespace:
 +
 --
-[source]
+[source,yaml]
```
Comment (nickboldt) [inline modules/installation/proc-rhdh-deploy-aks-operator.adoc]:
```suggestion
[source,terminal]
```

File: modules/installation/proc-rhdh-deploy-aks-operator.adoc [hunk 4]
```diff
@@ -160,7 +160,7 @@ kubectl -n <your_namespace> apply -f rhdh.yaml
 . Optional: To delete the CR, run the following command:
 +
 --
-[source]
+[source,yaml]
```
Comment (nickboldt) [inline modules/installation/proc-rhdh-deploy-aks-operator.adoc]:
```suggestion
[source,terminal]
```

File: modules/release-notes/con-release-notes-overview.adoc [hunk 5]
```diff
@@ -41,15 +41,15 @@ You must wait while the resources in the older versions are deleted and newer ve
...
+[source,terminal]
 ----
 oc login -u <user> -p <password> https://api.<HOSTNAME>:6443
 oc project devhub
 ----
 
 . For a new version of the {product-short} helm chart, run the following upgrade command:
 +
-[source,subs="attributes+"]
+[source,yaml,subs="attributes+"]
```
Comment (nickboldt) [inline modules/release-notes/con-release-notes-overview.adoc]:
```suggestion
[source,terminal,subs="attributes+"]
```

File: modules/upgrade/proc-upgrade-rhdh-helm.adoc [hunk 6]
```diff
@@ -29,15 +29,15 @@ It might take a few minutes to delete the resources in the older versions and to
...
+[source,terminal]
 ----
 oc login -u <user> -p <password> https://api.<HOSTNAME>:6443
 oc project <your_namespace>
 ----
 
 . For a new version of the {product-short} Helm chart, run the following upgrade command:
 +
-[source,subs="attributes+"]
+[source,yaml,subs="attributes+"]
```
Comment (nickboldt) [inline modules/upgrade/proc-upgrade-rhdh-helm.adoc]:
```suggestion
[source,terminal,subs="attributes+"]
```

**Comment by jmagak** [comment]
Closing this PR due to a conflict in the repository.

**Pull request closed without merging**